Library/Sender.ts

```diff
@@ -205,8 +216,19 @@ class Sender {
         }
     }
 
+    private _handleRedirect(location: string) {
+        this._numConsecutiveRedirects++;
+        // To prevent circular redirects
+        if (this._numConsecutiveRedirects < 10) {
+            this._redirectedHost = location;
+        }
+        else {
+            //TODO: Add ?redirect=false to avoid further redirection when 2.1 endpoint available
```

Comment:
still following up with Oleg on this if Breeze will support ?redirect=false.  Based on our last conversation, looks like Stamp Specific endpoints donot support ?redirec=false with AAD authentication enabled.